types/ember__object/tsconfig.json

```diff
@@ -15,8 +15,31 @@
             "../"
         ],
         "paths": {
+            "@ember/application": ["ember__application"],
+            "@ember/application/*": ["ember__application/*"],
+            "@ember/array": ["ember__array"],
+            "@ember/array/*": ["ember__array/*"],
+            "@ember/component": ["ember__component"],
+            "@ember/component/*": ["ember__component/*"],
+            "@ember/controller": ["ember__controller"],
+            "@ember/debug": ["ember__debug"],
+            "@ember/debug/*": ["ember__debug/*"],
+            "@ember/engine": ["ember__engine"],
+            "@ember/engine/*": ["ember__engine/*"],
+            "@ember/error": ["ember__error"],
             "@ember/object": ["ember__object"],
-            "@ember/object/*": ["ember__object/*"]
+            "@ember/object/*": ["ember__object/*"],
```

Comment:
This doesn't make a whole lot of sense to me. `@ember/object` is a low-level package that `@ember/engine` depends on. My hope was to introducing all of these additional entanglements.

Reply:
The test files import 'ember' which brings in all these dependencies. An alternate fix would be to change the tests to test just this package, if you want to work on that.

Reply:
proper fix is here: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/29761